### PR TITLE
Rewrite the Getting Started section

### PR DESCRIPTION
**Pull request by mairas**
The new text leans heavily on the project template repo and streamlines the process. Material unrelated to Getting Started has been removed. (Possibly to be re-added later on in a more suitable location.)

### CODE REVIEW

File: docs/pages/getting_started/index.md [hunk 1]
```diff
@@ -6,142 +6,157 @@ nav_order: 20
...
+SensESP device on the dashboard:
 
-If your Signal K Server has security enabled (it does by default), you should see an access request for your ESP in the Signal K Dashboard, under Security - Access Requests. (You must be logged into the Signal K Server to see the Security sub-menu.) Set the "Authentication Timeout" field to "NEVER", set the Permission to "Read / Write", then Approve it.
+![Dashboard view](assets/dashboard_view.png "Dashboard view"){:width="50%"}
 
-You should start seeing data from your ESP on the Signal K Data Browser (on the Server's main menu). You can also see lots of activity in the Visual Studio Code Serial Monitor, including the connection to the Signal K Server, the request for and approval of the security token, and the flow of data. Of course, the data is not likely to be valid, since you probably haven't yet connected anything to your ESP32. That's OK - if you've got data flowing at this point, you're ready to move on.
+Temperature readings on the data browser:
```
Comment (ba58smith) [inline docs/pages/getting_started/index.md:109]: Temperature? I don't see any other reference to a temperature sensor.

File: docs/pages/getting_started/index.md [hunk 2]
```diff
@@ -6,142 +6,157 @@ nav_order: 20
...
-6. Select a board. This refers to the microcontroller you'll be using. If you type `esp32` into the "Board" field, the list will show all the ESP32 boards that PIO works with. Select your specific board, unless you don't see it in the list, in which case select "Espressif ESP32 Dev Module".
-7. The "Framework" field should change automatically to "Arduino". Leave it that way.
-8. Click the "Finish" button, and your new Project will be created.
-9. A new `platformio.ini` file should open in the editor, and in the file navigator, if you click on "src", you'll see that there is a `main.cpp` file. Double-click on that to open it in the editor, and now you'll be looking at the only two files you should need to work with to create your first SensESP project.
+## Building and Installing the Project Template
 
-## Getting a Good platformio.ini File
+Open the project template directory in Visual Studio Code.
+VSCode should automagically detect that the directory as a PlatformIO project and recommend installing it.
```
Comment (ba58smith) [inline docs/pages/getting_started/index.md:55]: "is a PlatformIO project" (not "as").

File: docs/pages/getting_started/index.md (hunk 1 above)
Comment (mairas) [reply docs/pages/getting_started/index.md:109]: Good catch; I copied the screenshot parts from a SH-ESP32 tutorial and didn't notice that.

File: docs/pages/getting_started/index.md (hunk 2 above)
Comment (mairas) [reply docs/pages/getting_started/index.md:55]: Fixed.

**Pull request merged**